app/src/main/java/chat/rocket/android/helper/Constants.kt

```diff
@@ -16,7 +16,7 @@ object Constants {
     const val WIDECHAT = true
     const val WIDECHAT_DEV = false
 
-    const val DEEP_LINK = "deep_link"
+    const val DEEP_LINK_INFO = "deep_link_info"
```

Comment:
Do you ever clear this value after it's been used the first time?

Reply:
This is a constant string.